flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -312,4 +319,41 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
 
         result(item)
     }
+
+    private var totalFrames: UInt = 0
+    private var frozenFrames: UInt = 0
+    private var slowFrames: UInt = 0
+
+    private func beginNativeFrames(result: @escaping FlutterResult) {
+      guard PrivateSentrySDKOnly.isFramesTrackingRunning else {
+        print("Native frames tracking not running.")
+        result(nil)
+        return
+      }
+
+      let currentFrames = PrivateSentrySDKOnly.currentScreenFrames
+      totalFrames = currentFrames.total
+      frozenFrames = currentFrames.frozen
+      slowFrames = currentFrames.slow
+
+      result(nil)
+    }
+
+    private func endNativeFrames(result: @escaping FlutterResult) {
+      guard PrivateSentrySDKOnly.isFramesTrackingRunning else {
+        print("Native frames tracking not running.")
+        result(nil)
+        return
+      }
+
+      let currentFrames = PrivateSentrySDKOnly.currentScreenFrames
+
+      let item: [String: Any] = [
+          "totalFrames": currentFrames.total - totalFrames,
+          "frozenFrames": currentFrames.frozen - frozenFrames,
+          "slowFrames": currentFrames.slow - slowFrames,
```

Comment:
Let's check if they are not zeroed before returning the map.
https://github.com/getsentry/sentry-react-native/blob/main/ios/RNSentry.m#L226-L234